scaletest/terraform/coder.tf

```diff
@@ -242,6 +245,168 @@ resource "local_file" "kubernetes_template" {
   EOF
 }
 
+# TODO(cian): Remove this when we have support in the Helm chart.
+# Ref: https://github.com/coder/coder/issues/8243
+resource "local_file" "provisionerd_deployment" {
+    filename = "${path.module}/../.coderv2/provisionerd-deployment.yaml"
+    content = <<EOF
```

Comment:
curious: can this be `source`d from a regular YAML file?

Reply:
I think you can include it as a template, but I find it more straightforward to just inline.

Reply:
👍 

My only concern is IDE support, but that is not affecting the product.